src/main/java/com/questhelper/helpers/quests/deathontheisle/DeathOnTheIsle.java

```diff
@@ -0,0 +1,734 @@
+
+		var talkToCostumerAboutStainedCostume = talkToCostumerAgain.copy();
+		talkToCostumerAboutStainedCostume.addDialogStep("It seems one of the costumes has a stain on it.");
+		talkToCostumerAgain.addSubSteps(talkToCostumerAboutStainedCostume);
+		var talkedAboutStainedCostume = new VarbitRequirement(11256, 1);
+
+		var talkToCostumerAboutHiddenPassage = talkToCostumerAgain.copy();
+		talkToCostumerAboutHiddenPassage.addDialogStep("Did you know there was a hidden passage in here?");
+		talkToCostumerAgain.addSubSteps(talkToCostumerAboutHiddenPassage);
+		// var talkedAboutHiddenPassage = new VarbitRequirement(11254, 1);
+
+		investigateTheatreCellar = new ConditionalStep(this, talkToCostumerAboutHiddenPassage);
+		investigateTheatreCellar.addStep(not(inVilla), returnToButlerAndHeadInside);
+		investigateTheatreCellar.addStep(aroundVilla, climbFirstLooseRocksToTheatre);
+		investigateTheatreCellar.addStep(outsideVillaByLooseRocks, climbSecondLooseRocksToTheatre);
+		investigateTheatreCellar.addStep(not(inTheatreCellar), enterBackstage);
+		investigateTheatreCellar.addStep(not(searchedCrateNextToStairs), searchCrateNextToStairs);
+		investigateTheatreCellar.addStep(not(searchedBookshelf), searchBookshelf);
+		investigateTheatreCellar.addStep(not(searchedCostumeRack), searchCostumeRack);
+		investigateTheatreCellar.addStep(not(talkedAboutActors), talkToCostumerAboutActors);
+		investigateTheatreCellar.addStep(not(talkedAboutPoison), talkToCostumerAboutPoison);
+		investigateTheatreCellar.addStep(not(talkedAboutStainedCostume), talkToCostumerAboutStainedCostume);
+
+		/// 40
+		var climbUpFromTheatreCellar = new ObjectStep(this, ObjectID.STAIRS_54717, new WorldPoint(1469, 9328, 0), "Climb up the stairs of the theatre cellar.");
+
+		// TODO: can we only highlight "More options" if "Naiatli" is not one of the options?
+		speakToGuards = new NpcStep(this, NpcID.STRADIUS, new WorldPoint(1472, 2925, 0), "Report back to Stradius near the theatre and accuse Naiatli.");
+		speakToGuards.addDialogStep("More options...");
+		speakToGuards.addDialogStep("Naiatli.");
+		speakToGuards.addSubSteps(climbUpFromTheatreCellar);
+
+		// varbit 11257=1 if spoken to stradius a bit
+
+		snitchToGuardsStep = new ConditionalStep(this, speakToGuards);
+		snitchToGuardsStep.addStep(not(inVilla), returnToButlerAndHeadInside);
+		snitchToGuardsStep.addStep(aroundVilla, climbFirstLooseRocksToTheatre);
+		snitchToGuardsStep.addStep(outsideVillaByLooseRocks, climbSecondLooseRocksToTheatre);
+		snitchToGuardsStep.addStep(inTheatreCellar, climbUpFromTheatreCellar);
+
+		// 42 + 45
+		talkToStradiusToEnterTheTheatre = new NpcStep(this, NpcID.STRADIUS, new WorldPoint(1472, 2925, 0), "Talk to Stradius to enter the theatre.");
+		confrontNaiatli = new NpcStep(this, NpcID.NAIATLI, new WorldPoint(1465, 2932, 0), "Confront Naiatli, attacking her when she runs away. When Clodius appears, attack him instead. Use Protect from Melee to avoid all damage.");
```

Comment:
From what I could tell they never get to attack you, so no need for protect from melee? Not sure if maybe I had 1 strength I wouldn't hit enough to instant-transition maybe, but the attacks felt scripted.

Reply:
The attacks did feel very scripted to me too - only the Adala "fight" took more than one hit

Reply:
I removed any mention of Protect from Melee in the Naiatla/Clodius fight in [`b79e84b` (#1774)](https://github.com/Zoinkwiz/quest-helper/pull/1774/commits/b79e84bfc69258b4460cbea6d19b4e919610d035)

Reply:
Changed the combat requirements to just say "2 unlosable fights"
You do get combat experience, so I don't want to remove the combat requirements entirely, and I'm not sure I'm confident enough to say: "if you turn off auto retaliate you can avoid getting combat experience"